packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -46,135 +53,105 @@
             "$ref": "#/definitions/backgroundImage",
             "description": "Image to display in the background of this page. It should be a horizontal image and it will be fixed at the bottom of the page."
           },
-          "layout": {
-            "type": "object",
-            "description": "The layout specification for the page.",
-            "additionalProperties": false,
-            "required": ["type"],
+          "navigation": {
+            "$ref": "#/definitions/navigation",
+            "description": "Configure the node in the navigation panel for this page."
+          },
+          "guidancePanel": {
+            "$ref": "#/definitions/guidancePanel",
+            "description": "The guidance panel specification for this page."
+          },
+          "layout": { "doNotSuggest": true },
+          "header": { "doNotSuggest": true },
+          "groups": { "doNotSuggest": true }
+        },
+        "oneOf": [
+          {
+            "required": ["type", "layout"],
             "properties": {
               "type": {
-                "type": "string",
-                "description": "Layout type",
-                "enum": ["SingleColumn", "TwoColumn"],
-                "enumDescriptions": [
-                  "A page layout with a single panel of items.",
-                  "A page layout with left and right panels of items."
-                ]
+                "const": "Configuration"
               },
-              "header": {
```

Comment:
I moved header down to the `definitions` in the schema so it can referenced for validation page and for the single/2column layouts